packages/sitecore-jss/src/graphql-request-client.test.ts

```diff
@@ -169,6 +173,31 @@ describe('GraphQLRequestClient', () => {
   });
 
   describe('Working with retryer', () => {
+    it('should use the clientConfig values configured by the client', () => {
```

Comment:
This unit test doesn't really verify the initial problem. For example, LayoutService passed retryStrategy to GraphQLRequestClient. I would expect to implement unit tests for LayoutService, DictionaryService.  This unit test is redundant, it checks javascript class functionality

Reply:
I have added test for each service.